examples/menubar/css/menubar-navigation.css

```diff
@@ -92,23 +130,61 @@
   background-repeat: repeat-x;
 }
```

Comment:
We can't use the following lines for examples because relative paths from CSS files do not work in Codepen:
```
  background-image: url("../images/separator.svg");
```
 What I've been doing is copying the SVG from the file into this URL, like in #1651 
```
url("data:image/svg+xml,%3C%3Fxml version='1.0' encoding='utf-8'%3F%3E%3Csvg xmlns='http://www.w3.org/2000/svg' width='12' height='12' viewBox='0 0 12 12'%3E%3Cline x1='0' y1='6' x2='12' y2='6' style='stroke:black;stroke-width:1' /%3E%3C/svg%3E%0A");
```
I used this URL encoder to do it: https://yoksel.github.io/url-encoder/

Can you make that change here? The other option is to define the SVG in the HTML then reference it in the HTML, or have a relative path from the HTML to the SVG.

Reply:
Fixed